src/main/java/net/sf/jabref/gui/specialfields/SpecialFieldIcon.java

```diff
@@ -0,0 +1,75 @@
+package net.sf.jabref.gui.specialfields;
+
+import javax.swing.Icon;
+import javax.swing.JLabel;
+
+import net.sf.jabref.gui.IconTheme;
+import net.sf.jabref.logic.specialfields.SpecialField;
+import net.sf.jabref.model.entry.SpecialFieldValue;
+
+
+public class SpecialFieldIcon {
+
+    public static Icon getRepresentingIcon(SpecialField field) {
+        if (SpecialField.PRINTED.equals(field)) {
```

Comment:
Heh, I was expecting that. Why not turn the whole thing into a `HashMap` instead of an if/else or a switch? I have no clue if that is preferable over the other alternatives, though.

Side remark: This would not be switching on a String, but on an enum value.